tests/aws/services/lambda_/functions/common/echo/dotnet6/Makefile

```diff
@@ -1,9 +1,20 @@
+UNAME := $(shell uname -m)
+ifeq ($(UNAME),x86_64)
```

Comment:
I wished this could be shorter but Ubuntu (in CI) yields `aarch64` and macOS yields `arm64`, so we need the conditional to standardize the `UNAME` architecture  :(